ompi/mca/coll/acoll/README

```diff
@@ -0,0 +1,15 @@
+Copyright (c) 2023-2024 Advanced Micro Devices, Inc. All rights
```

Comment:
have you thought about what needs to be done to extend this for multiple nodes?

Reply:
Some of the APIs (like bcast, barrier, allgather) support multi-node case. However, it is not extensively tested for multi-node, we will test them and extend other APIs also to multi-node.